README.md

```diff
@@ -42,3 +40,28 @@ https://annu12340.github.io/Javascript-Notes/
 ### CSS Notes
 
 https://github.com/ResourceAggregator/Frontend-stuff/blob/master/css/css.md
+
+
+
+
+                                                        NEGATIVE_INFINITY AND POSITIVE_INFINITY IN JAVASCRIPT 
```

Comment:
Add this topic in beginner ' section after DOM manipulation

Reply:
but there is no DOM Manipulation in beginner section.

Reply:
Yes there is
Have a look once more
![Screenshot1](https://user-images.githubusercontent.com/43414928/94900346-734c4480-04b2-11eb-9065-e06d3c7de449.png)